missilewars-plugin/src/main/java/de/butzlabben/missilewars/listener/PlayerListener.java

```diff
@@ -153,6 +157,34 @@ public void onLeaveWithStuff(PlayerArenaLeaveEvent event) {
         MissileWars.getInstance().getSignRepository().getSigns(game).forEach(MWSign::update);
     }
 
+    // cancel anvil interaction to block renaming, and thus changing missiles
+    @EventHandler
+    public void onClick(InventoryClickEvent event) {
+        Game game = getGame(event.getWhoClicked().getLocation());
+        if (game != null) {
+            if (event.getInventory().getType() == InventoryType.ANVIL) {
```

Comment:
Also use guard clauses, just as in all other events

Reply:
I down't reworked this event in the PlayerListener. Only the "event" variable. Reworking coming with the next PR.